eureka-core/src/main/java/com/netflix/eureka/registry/rule/DefaultStatusRule.java

```diff
@@ -0,0 +1,30 @@
+package com.netflix.eureka.registry.rule;
+
+import com.netflix.appinfo.InstanceInfo;
+import com.netflix.eureka.lease.Lease;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is the default rule. It should be at the bottom of each rule chain.
+ * It matches always and returns the current status of the instance.
+ *
+ * Created by Nikos Michalakis on 7/13/16.
+ */
+public class DefaultStatusRule implements InstanceStatusOverrideRule {
```

Comment:
Rename to something like AlwaysMatchStatusRule to better describe what this rule does?